src/appmixer/cloudflare/IPListModel.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = context => {
+
+    /** Model for component SetBlockIPRule. Collection name: pluginAppmixerImpervaBlockIPRules */
+    class IPListModel extends context.db.Model {
+
+        static get collection() {
+
+            return 'ipList';
+        }
+
+        static get idProperty() {
+
+            return 'id';
+        }
+
+        static get properties() {
+
+            return [
+                'id',
+                'ip',
+                'removeAfter',
+                'auth',
+                'mtime'
```

Comment:
`mtime` is added automatically by the engine. You don't have to specify it here.